rh-tictactoe/src/app/screens/Login/index.js

```diff
@@ -34,7 +40,8 @@ const Login = ({ handleSubmit, loginFail }) => (
         <button className={Styles.submitButton} type="submit">
           Log in
         </button>
-        {loginFail ? <h3>User or password incorrect</h3> : null}
+        <SpinnerWithLoading isLoading={isLoading} />
+        <HeaderWithLoginFail loginFail={loginFail} />
```

Comment:
this is not how the HOC msut work. The HOC must surround the component. You are adding 2 new components this way